h/static/scripts/directive/test/excerpt-test.js

```diff
@@ -2,10 +2,24 @@
 
 var angular = require('angular');
 
-var assign = require('core-js/modules/$.object-assign');
 var util = require('./util');
 var excerpt = require('../excerpt');
 
+/**
+ * Wait for an <excerpt> to recompute its overflowing state.
+ *
+ * This happens asynchronously after an <excerpt> is created in order to wait
+ * for Angular directives used by the <excerpt>'s content to fully resolve.
+ *
+ * @return {Promise}
+ */
+function waitForLayout(element) {
+  element.scope.$digest();
+  return new Promise(function (resolve) {
+    window.requestAnimationFrame(resolve);
```

Comment:
In some places we do `require('raf')` for this. In other places we use `$$rAF` from angular internals. What's the status of the polyfill? Can we (perhaps in another PR) make this consistent across the client application?